athenet/utils/count_n_elts.py

```diff
@@ -0,0 +1,88 @@
+"""Function that for given neural network, returns lists of numbers of edges,
+weights and neurons per layer."""
+
+from  athenet.models import *
+from athenet.layers import *
+import numpy as np
+
+def _conv_ed(x, y, fx, fy, sx, sy, n_in, n_out, g=1):
```

Comment:
reading this, I have no idea what this function does - maybe a little longer names like filter_x ?